modules/ROOT/pages/objc.adoc

```diff
@@ -127,17 +127,14 @@ include::{partialsdir}/ios-framework-size.adoc[]
 
 == Supported Versions
 
-[width="70%"]
+[%autowidth.stretch]
 |===
 |Platform |Minimum OS version
 
 |iOS
-|10.0
-
-9.0 [DEPRECATED]
-
+|*10.0* (9.0 DEPRECATED
 |macOS
-|10.9
+|*10.11* (10.9 and 10.10 DEPRECATED)
```

Comment:
Could we move the deprecation notice for 10.10 and 10.9 to the note below and also include it in the release notes?

Reply:
Thanks @amarantha-k . Corrected in commit ebf7416a80ce56d8d2a7e7b14f30f7a84ced92fe.